autogluon/training/docker/0.3/py3/Dockerfile.cpu

```diff
@@ -101,7 +101,7 @@ RUN pip install --no-cache-dir -U \
 && pip uninstall -y jupyter-client jupyter-core jupyter-packaging jupyter-server jupyter-server-proxy \
     jupyterlab jupyterlab-nvdashboard jupyterlab-pygments jupyterlab-server jupyterlab-widgets ipywidgets \
     widgetsnbextension traitlets pydeck panel nbformat nbconvert nbclient pyppeteer matplotlib-inline \
-    ipython-genutils ipython ipykernel torchvision bokeh plotly notebook datashader
+    ipython-genutils ipython ipykernel bokeh notebook datashader
```

Comment:
Please break these package names into multiple lines, with one package name per line. It is currently hard to understand what packages are being installed/uninstalled, and whether any package names are repeated.

Also, why is `bokeh` being uninstalled?

Why does this image pass the https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/sanity/test_utility_installation.py#L62-L65 test if `bokeh` is not installed in the image?

Reply:
All those packages are uninstalled to make an image as small as possible. 
Re 1 package at 1 string - I understand readability is not prefect. But do you really prefer 40 strings instead of 5?
What's wrong with test_utility_packages_using_import? It checks only SM version.